R/plotFeatureLoadings.R

```diff
@@ -0,0 +1,404 @@
+#' Plot tree with feature loadings after performing a reducedDim (e.g. PCA, LDA, NMF)
+#'
+#' Based on the \code{\link[diffTop:plotASVcircular]{plotASVcircular}} method using phyloseq 
+#' and has been converted to use TreeSummarizedExperiment objects.
+#'
+#' @param object a
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'   object.
+#' 
+#' @param method One dimensionality reduction method of \code{c("PCA", "LDA", "NMF")}
```

Comment:
Do we need this if the method should also support general numeric loading matrix?

At least this should be optional. Ideally, the method is agnostic to which method generated the loadings.

Reply:
`method` could be `dimred`. You can check plotRDA which plots the principal coordinates

Reply:
For now it doesn't recognize the reduction method used but it will probably be removed once it is done

Reply:
Ok - can we remove this and resolve the conversation, or is this needed?

Reply:
One problem is that in practice the ordination result can be named abitrarily (it can be "pca" or "pca_something"). There should be a way to determine the fetched ordination from reducedDim.

The result should be calculated with runPCA or addLDA, so the result is standardized. If the loadings are not in certain place, you can just give error.

See plotRDA

Reply:
Yes. 

And loadings are always a numeric matrix. If we can support arbitrary numeric matrices in addition to TreeSE slot here then that would be worth having I think.